pkg/daemon/update.go

```diff
@@ -379,6 +379,9 @@ func (dn *CoreOSDaemon) applyOSChanges(mcDiff machineConfigDiff, oldConfig, newC
 			MCDPivotErr.WithLabelValues(nodeName, newConfig.Spec.OSImageURL, err.Error()).SetToCurrentTime()
 			return err
 		}
+		dn.recorder.Eventf(getNodeRef(dn.node), corev1.EventTypeNormal, "OSUpgradeApplied", "OS upgrade applied; new MachineConfig (%s) has new OS (%s)", newConfig.Name, newConfig.Spec.OSImageURL)
+	} else {
+		dn.recorder.Eventf(getNodeRef(dn.node), corev1.EventTypeNormal, "OSUpgradeSkipped", "OS upgrade skipped; new MachineConfig (%s) has same OS (%s) as old MachineConfig (%s)", newConfig.Name, newConfig.Spec.OSImageURL, oldConfig.Name)
```

Comment:
same here
```suggestion
		dn.nodeWriter.Eventf(corev1.EventTypeNormal, "OSUpgradeSkipped", "OS upgrade skipped; new MachineConfig (%s) has same OS (%s) as old MachineConfig (%s)", newConfig.Name, newConfig.Spec.OSImageURL, oldConfig.Name)
```

Reply:
Also, instead of "same OS ", it should be OS image

Reply:
Agreed. That sounds better.